collections/Laud_Misc/MS_Laud_Misc_204.xml

```diff
@@ -37,44 +36,486 @@
                      <idno type="SCN">799</idno>
                   </altIdentifier>
                </msIdentifier>
+               <!-- https://www.christies.com/lot/lot-book-of-hours-use-of-sarum-5949404/?from=salesummery&intobjectid=5949404&sid=44abae07-ddfe-4725-8fe1-d143ce8b44dc
+               https://www.jstor.org/stable/41154814?seq=1
+               -->
+               <head>Book of Hours, Use of Sarum; Flanders (?Bruges), 15th century, second
+                  quarter</head>
                <msContents>
-                  <msItem class="#liturgica" n="1" xml:id="MS_Laud_Misc_204-item1">
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem class="#liturgica" xml:id="MS_Laud_Misc_204-item1">
+                     <locus>(fols. 4r-142r)</locus>
                      <title key="work_10579" type="desc">Book of Hours, Use of Sarum</title>
-                     <textLang mainLang="la">Latin</textLang>
+                     <msItem>
+                        <note>For added texts on pastedown and fols. 2r-3v see below, 'Added texts'. Laudian (?)
+                           table on contents on fol. 3v.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 4r-9r)</locus>
+                        <title type="desc">Fifteen O’s</title>
+                        <incipit>O domine Iesus Christe eterna dulcedo te amancium iubilus
+                           excedens</incipit>
+                        <explicit>te merear laudare cum omnibus sanctis tuis in eternum. Amen.</explicit>
+                        <note>Ps.-Bridget of Sweden; pr. <title>Revelationes selectae S. Birgittae</title>, ed. A. Heuser (1851), 3442-51. Fol. 4r was originally blank (fol. 4v is a miniature); text was added in England
+                           in the later 15th or the 16th century to supply the loss of the
+                           first 'O' on the folio originally preceding fol. 5.</note>
+                        <note>Fol. 9v originally blank, for added prayer see below, 'Added texts'.</note>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 10v/11r-23v)</locus>
```

Comment:
Does this start on 10v or 11r? (Is this something needing to be checked or a reference style whose meaning I am missing?)